src/meta/src/hummock/metrics_utils.rs

```diff
@@ -45,6 +45,7 @@ pub fn trigger_version_stat(
     metrics
         .max_committed_epoch
         .set(current_version.max_committed_epoch as i64);
+    // TODO: this op may be costly
```

Comment:
Indeed we've seen version of tens of megabytes. Will any fix include in this PR?

Reply:
I have changed to implement an `estimated_encoded_len` by ourselves.